egs/wsj/s5/local/wsj_word_class_prep.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+#
+# Copyright 2010-2016 FAU Erlangen  (Author: Axel Horndasch).  Apache 2.0.
+
+# To be run from one directory above this script.
+#
+# the main purpose of the script is to copy / convert word class files
+# from an external source (${CORPUS_WORD_CLASS_DIR}) to the place in
+# the Kaldi directory structure (${KALDI_WORD_CLASS_DIR})
+
+export LC_ALL=C
+
+CORPUS_NAME=wsj
+
+if [ $# != 2 ]; then
+  echo "Usage: ./local/${CORPUS_NAME}_word_class_prep.sh /path/to/corpus/word/class/files new/kaldi/word/class/dir"
```

Comment:
In this usage comment or at the top of this script (or both) there should be a bit more explanation of what this script does, and what its input and output formats are.
There was the same issue with at least one of the perl scripts you added, that there seemed to be too little explanation of what the script was for and what it does.